client/stylesheets/_toCat.scss

```diff
@@ -3937,7 +3939,8 @@ On click the images will be displayed at normal size to complete the effect
     height: 220px;
     display: inline-block;
     margin: 0.6%;
-    border: 2px solid #00bfff;
+    border: 1px solid #00bfff;
+    border-radius: 4px !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces